CMakeLists.txt

```diff
@@ -378,7 +416,10 @@ else ()
   set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} "${ENSMALLEN_INCLUDE_DIR}")
 endif ()
 
-find_package(cereal "${CEREAL_VERSION}" REQUIRED)
+if(NOT CMAKE_CROSSCOMPILING)
+  find_package(cereal "${CEREAL_VERSION}" REQUIRED)
+endif()
```

Comment:
Since cereal is header only too, I think it should be reasonable to use whatever the user has available on the system.

Reply:
Same as above :(